src/main/java/io/github/thebusybiscuit/sensibletoolbox/utils/WordUtils.java

```diff
@@ -0,0 +1,63 @@
+package io.github.thebusybiscuit.sensibletoolbox.utils;
+
+public class WordUtils {
+
```

Comment:
Around here, add a private constructor with no arguments. Normally you would also throw an exception also, but as that's not the case for this plugin's other utility classes we can ignore that for now. Check out `VanillaInventoryUtils` to see what I mean here.